backend/src/db/migrations/6-blocks.sql

```diff
@@ -27,7 +27,9 @@ create table blocks (
 
     -- All videolist-like blocks
     videolist_layout video_list_layout,
-    videolist_order video_list_order
+    videolist_order video_list_order,
+
+    video_id bigint references events on delete restrict
```

Comment:
Please add a "section comment", as otherwise this belongs to the "All videolist-like blocks" section.